main.go

```diff
@@ -25,6 +27,7 @@ var AnalyticsStore storage.AnalyticsStorage
 var UptimeStorage storage.AnalyticsStorage
 var Pumps []pumps.Pump
 var UptimePump pumps.MongoPump
+var PumpsTimeout sync.Map
```

Comment:
sync.Map not really needed here. Can be replaced by Pump field.

Reply:
Added SetTimeout and GetTimeout to pump interfaces + timeout var in each pump.